docs/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM squidfunk/mkdocs-material:8.3.9
+FROM squidfunk/mkdocs-material:9.4.6
```

Comment:
`squidfunk/mkdocs-material:9.4.6` (new version) also returns error for `PyYAML 6.0.0`.
Updated it to avoid conflicts with new `PyYAML` version other packages after subsequent `Python` updates.